temporalcli/commands.workflow.go

```diff
@@ -7,13 +7,14 @@ import (
 
 	"github.com/fatih/color"
 	"github.com/google/uuid"
-	"github.com/temporalio/cli/temporalcli/internal/printer"
 	"go.temporal.io/api/batch/v1"
 	"go.temporal.io/api/common/v1"
 	"go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/query/v1"
 	"go.temporal.io/api/workflowservice/v1"
 	"go.temporal.io/sdk/client"
+
+	"github.com/temporalio/cli/temporalcli/internal/printer"
```

Comment:
Why'd this move?

Reply:
order of imports is different for CLI and OSS. My IDE is configured for OSS correct order. I wonder if we can/should have  project specific configurations.

Reply:
Yeah, I am not a fan of the OSS approach that groups some non-stdlib imports separate from other non-stdlib imports. But it's not a big deal, it's pedantic.